docs/sourcecontrol/faq.md

```diff
@@ -65,6 +65,12 @@ One way to avoid these prompts is to set up a [credential helper](https://help.g
 
 Another option is to disable the auto fetch feature by changing the following setting: `"git.autofetch": false`.
 
+### Why is VS Code telling me that the git repository is potentially unsafe?
```

Comment:
Did we want to go with:
"Why is VS Code warning that the git repository is potentially unsafe?"

"telling" is a little anthropomorphic and also not inclusive to deaf users

Reply:
@gregvanl, no specific reason, and you are absolutely correct! 
Happy to rephrase if you have a better suggestion. Thanks!

Reply:
Cool, I'll merge and update the wording when I do an edit pass on vnext.